integration/containerd/confidential/agent_api.bats

```diff
@@ -4,56 +4,53 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-load "${BATS_TEST_DIRNAME}/lib.sh"
-load "${BATS_TEST_DIRNAME}/asserts.sh"
 load "${BATS_TEST_DIRNAME}/../../../lib/common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
-	start_date=$(date +"%Y-%m-%d %H:%M:%S")
-	sandbox_name="kata-cc-busybox-sandbox"
-	pod_config="${FIXTURES_DIR}/pod-config.yaml"
-	pod_id=""
-
-	echo "Delete any existing ${sandbox_name} pod"
-	crictl_delete_cc_pod_if_exists "$sandbox_name"
-
-	echo "Prepare containerd for Confidential Container"
-	SAVED_CONTAINERD_CONF_FILE="/etc/containerd/config.toml.$$"
-	configure_cc_containerd "$SAVED_CONTAINERD_CONF_FILE"
-
-	echo "Reconfigure Kata Containers"
-	clear_kernel_params
-	switch_image_service_offload on
-	enable_full_debug
+	setup_common
 
 	# Test will change the guest image so let's save it to restore
 	# on teardown.
 	saved_img="$(save_guest_img)"
+
+	agent_config_file="${FIXTURES_DIR}/agent-configuration-no-exec.toml"
+
+	# In case the tests run behind a firewall where images needed to be fetched
+	# through a proxy.
+	local https_proxy="${HTTPS_PROXY:-${https_proxy:-}}"
+	if [ -n "$https_proxy" ]; then
+		local new_file="${BATS_FILE_TMPDIR}/$(basename ${agent_config_file})"
+		echo "https_proxy = '${HTTPS_PROXY}'" > "$new_file"
```

Comment:
we may need use `https_proxy` instead of `HTTPS_PROXY`

Reply:
Good catch! I just updated this PR with the fix.